packages/assets-controllers/src/CollectiblesController.ts

```diff
@@ -1,19 +1,13 @@
 import { EventEmitter } from 'events';
```

Comment:
We should rename this controller before splitting up the monorepo. We wanted to change all of the "collectible" names to "NFT"

Reply:
Deferred to a future PR.